JASP-Engine/JASP/R/ttestonesample.R

```diff
@@ -343,9 +356,21 @@ TTestOneSample <- function(jaspResults, dataset = NULL, options, ...) {
   rowNo <- 1
   if (!ready) 
     variables = "."
+  
   for (variable in variables) {
     
     if (ready) {
+      errors <- .hasErrors(dataset, 
+                           message = 'short', 
+                           type = c('observations', 'variance', 'infinity'),
+                           all.target = variable,
+                           observations.amount = c('> 5000'))
```

Comment:
Where is this > 5000 coming from? I don't think we had this before

Reply:
I reverted the Normality error check to `observations.amount = c('< 3', '> 5000')` as it was previously

Reply:
But this is the descriptives plot?